packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -1015,6 +1017,30 @@ const UsernamePasswordAuthenticationStrategyEditor = observer(
   },
 );
 
+const MiddleTierUsernamePasswordAuthenticationStrategyEditor = observer(
+  (props: {
+    authSpec: MiddleTierUsernamePasswordAuthenticationStrategy;
+    isReadOnly: boolean;
+  }) => {
+    const { authSpec, isReadOnly } = props;
+    return (
+      <>
+        <ConnectionEditor_StringEditor
```

Comment:
Would recommend adding a description for this input to guide better the users on what exactly this property does

Reply:
Added in this [commit](https://github.com/finos/legend-studio/pull/1238/commits/b8abe20c271749242cc1317bbe47b2d36624b939)